docs/microbit_micropython_api.rst

```diff
@@ -41,8 +41,8 @@ The LED display is exposed via the `display` object::
     display.get_pixel(x, y) # gets the brightness of the pixel (x,y)
     display.set_pixel(x, y, val) # sets the brightness of the pixel (x,y) to val (between 0 and 9, inclusive)
     display.clear() # clears the display
-    display.print(string, delay=400) # prints the string to the display one character at a time
-    display.print(image, delay=400) # prints the image to the screen.
+    display.show(string, delay=400) # prints the string to the display one character at a time
```

Comment:
@deshipu It would be good to remove "prints" on line 44 as well. Perhaps "Outputs the string to the display in the following pattern: output one character, wait the delay time, clear display, output the next character."